server/db/db-connection.js

```diff
@@ -0,0 +1,124 @@
+import dotenv from 'dotenv';
+import { Pool } from 'pg';
+
+
+dotenv.config();
+
+class Database {
+  constructor() {
+    this.pool = new Pool({
+      user: 'epic_mail',

```

Comment:
Please include these into your .env file `user, host, database, password, port`

Reply:
I am facing a challenge, tomorrow I will ask help to the colleagues to fix it!